### ISSUE TITLE
Desperately needs help: Epic Game Launcher does not launch at all

### ISSUE THREAD

**Issue by tsunamilx** [moraroy/NonSteamLaunchers-On-Steam-Deck#513]
Hey guys, 

I've been struggling for this for a couple of days now, I only installed Epic Game Launcher, the launcher appears in the Steam library but it does not launch at all, I click "start", it says "launching", and after a couple of seconds, it back to "start".

I searched a lot and tried all the things I can find:
- adding '-opengl' (automatically added by NSL)
- tried verious proton version: GE-Proton 8 and 9, Proton 8.0.5, Proton Experimental
- reinstall NSL, both desktop version and decky plugin version, I dont see any error during install
- restart deck

Nothing works for me... I am desperate right now please help...

If there is a log file somewhere that can help troubleshotting, please let me know, and I will upload here.

Thanks.

**Comment by moraroy** [comment]
make sure you have the latest version of everything this still an issue? @tsunamilx

**Comment by tsunamilx** [comment]
Yes. I feel it might be something specific to my device, maybe I installed something that somehow causes the problem. Is there any log file that can help troubleshooting?

**Comment by moraroy** [comment]
@tsunamilx is compatability being forced in the properties?

**Comment by moraroy** [comment]
@tsunamilx is anything even installed? Open up NonSteamLaunchers.desktop does the window show any paths? If so that means your launchers are correctly installed. The only think I can think of is deleting the shortcuts and having NSL "remake them".

**Comment by tsunamilx** [comment]
I will try clear everything and reinstall NSL to remake them, will let you know later, thanks.

**Comment by tsunamilx** [comment]
It worked... kind of ...
Epic Game Launcher launches, but cannot pass the login screen, it crashes right after login, it says "An Unreal process has crashed:  UE4-EpicGamesLauncher"

**Comment by lostprophet** [comment]
Same crash for me on the Steam Deck OLED latest stable 3.6.20.
I get to the main page, input my credentials, try to log in.
Sometimes I get a completely black screen (I can close the whole thing by pressing the STEAM button and closing Epic), or get the crash that tsunamilx described.

Let me know if any info is needed. Note, I have never used NonSteamLauncher on this Deck, this is the first time using. I only tried the Decky plugin though.

**Comment by Valekj86** [comment]
![IMG_20241230_220551](https://github.com/user-attachments/assets/d58cd40c-8b99-4aed-b342-2b1d3c373d52)

Hello, i have the same problem. Steam Deck 1  TB Oled. Newly installed at 24.12.2024. The only other luncher I have is heroic luncher and junk store.

**Comment by moraroy** [comment]
@Valekj86 did you try a different Proton?

**Comment by saschanaz** [comment]
The script downloaded and set GE-Proton9-22 as the default, with which Epic Launcher crashes, and same for Proton Hotfix, Proton Experimental, Proton 9.0-4, Proton 8.0-5. Did not try lower versions.

**Comment by FEEEENGYM** [comment]
@Valekj86 Have you solved this problem?
 I also encountered the same problem.

**Comment by diggles4** [comment]
Anyone having luck? Same issue here.

**Comment by saschanaz** [comment]
Tried reinstalling and the client now updates with infinite loop (update finishes, client restarts, and then the update starts again)

**Comment by moraroy** [comment]
@FEEEENGYM @diggles4 @saschanaz wow I hope this isn't a new issue for Epic :/ don't have a fix yet I'm sorry, unfortunately when I install Epic it installs fine. Do you guys all have the oled?

**Comment by diggles4** [comment]
OLED here, yes.

**Comment by smcnamee22** [comment]
I'm in a similar position. It was all fine, then a week or 2 ago the client had an update, it crashed after finishing the update and wouldn't start again. I tried reinstalling (with and without a fresh start) and upon booting it tries to apply updates to complete the installation then immediately crashes at the end. and when i try to boot again it retries the updates then crashes again. every time now. @moraroy you saying its working for you makes me think there is something cached somewhere (that isn't being cleaned up from a fresh start) that is messing us up. do you have any idea what that could be? i may just reformat my oled and see if that fixes it.

**Comment by smcnamee22** [comment]
I reformatted and still the exact same behavior. I think it must have something to do with that new epic client update.

**Comment by saschanaz** [comment]
>all have the oled?

I have an LCD Steam Deck.

**Comment by moraroy** [comment]
@smcnamee22 did some more testing you guys are correct, it's definitely not working now, I had my launcher updated a couple days ago and it had seemed to ran fine until I guess this new update. In my testing, I noticed a couple things. In the error logs of "Appdata" the Epic Games launcher is not able to authenticate the Manifest files. Not sure if that's related but I think it might be because the launcher can't seem to get past finishing the "update" of the Epic Games services. I also noticed something else. It seems Epic Games Launcher either isn't installing all the way or they have stopped installing the win64 version of Epic. The reason why I mentioned this is because I was just now able to get my games working again by changing the folder name from "win32" to "win64" this would basically change the binary to a different .exe. however since I uninstalled just like you did, it now shows that that win64 binaries are no longer being installed by Epic Games launcher. This may be because it won't finish updating the manifest files like I said before. So I'm not sure what has changed but I think the best thing to do right now is wait and see if Epic fixes this or somehow do your best to "not update" the launcher if you're able to open it. If you have an older version of the launcher like I did. You can change the starting and target properties in the shortcut from "win32" to " win64" for all your games and shortcuts for the time being. That's only if you have that folder

**Comment by diggles4** [comment]
If it helps, using default Epic install option in desktop/Decky, I'm able to get to the login screen but it then goes to black (in game mode) or to the "An Unreal process has crashed" (desktop mode, see Valekj86's screenshot above).

If I install Epic using the "separate folder" option, it simply crashes after the "verifying update" progress bar. Same in desktop mode, and same if I use desktop installer instead of Decky.

**Comment by moraroy** [comment]
@diggles4 yeah I think their new update is broken, I think this also may be an issue to some users even on windows. For now we just have to wait and see.

**Comment by Cevap05** [comment]
> [@diggles4](https://github.com/diggles4) yeah I think their new update is broken, I think this also may be an issue to some users even on windows. For now we just have to wait and see.

Thanks for the mention. Read it all, guess will wait here and see if anyone mentions a fix or Epic makes an update.

**Comment by moraroy** [comment]
@Cevap05 I actually currently have mine fixed, am playing my games again..... thing is, I'm not sure how to go about fixing it for NSL. And apparently the Epic Games services doesn't work. So still working on it. But progress nonetheless!

**Comment by Cevap05** [comment]
@moraroy Nice! Wonder if you having a prior version to it has any impact? I'm doing a clean install, alas do get the UE crash window after login attempt. We'll see what comes of it I suppose. Do you suspect it has anything to do with NSL, or pretty much Epic end?

**Comment by moraroy** [comment]
@Cevap05 probably Epic because it broke once it updated. Epic has never had any issues  like this before.

**Comment by kuzeljko** [comment]
@moraroy I somehow made it work. Not sure what fixed it but here is what I did (Desktop mode):
In TARGET and START IN replaced win32 with win64
Removed  '-opengl' from TARGET
Switched to proton 7.0-6
Epic started and I got login screen
On login screen there is settings cog in bottom left corner
In settings I checked for updates
It crashed
Run it again, in settings says update available
Updated and it closed
Run it again, added login credentials, It WORKED
Switched to Proton Experimental, It is still working
Restarted SD, still working
Switched to Game mode, instaling and running games worked
It complained about some update services not being able to connect

Probably only updating from settings is what fixed it but I'm not sure. Not touching it now

SD updated to the latest in time of writing

Hope this helps somebody
If there are some logs or more info that I can provide, just let me know

**Comment by moraroy** [comment]
@kuzeljko good job brother, excellent work. Quick question, does changing it back "win32" in the shortcut allow it to work? The launcher also was trying to install Epic Online Services, which is needed for online functionality.

**Comment by kuzeljko** [comment]
@moraroy in game mode switched to win32, added -opengl flag, proton experimental, it still launches but I'm unable to install online services. Error code:EOS-ERR--2147483392

**Comment by Cevap05** [comment]
@kuzeljko Hey thanks for this comment. I did re-create your steps and want to add my input. Basically did everything up to the cog wheel where I saw an update and something along the lines of "Update and Restart". When I pressed it my window would immediately close and Epic would stop working after a short time. I then re-booted to see it "Update and Restart" again, so I clicked it. Same thing happened. I did this like another three times and it was always the same outcome. Never got a verification of the update being installed. Anyways I figured I'd just try to login. To my surprise I actually did!

I noticed the screen would flicker like crazy. I decided to close the app and restart, only changing the compatibility layer from "7.0-6" to Experimental. It logged me in again and the flickering was no more.

I also got a prompt to "Install Epic Online Services", which seemed to fail immediately. In any case, I'm trying to install a game right now and we'll see if it works. Installing as of this message. Thanks.

**Comment by kuzeljko** [comment]
Cool, we got somewhere. I'll play around tonight some more and report if I find something.

**Comment by moraroy** [comment]
@kuzeljko @Cevap05 just want to point out.... do not "uninstall" Epic Games launcher. You will no longer have the "win64" binary installed anymore due to the bug. if you try to re-install, it will make kuzeljko's fix/workaround not work.

**Comment by Cevap05** [comment]
Last thing before I head off. I installed a game. Then went to manual scan to get it into Steam. It did recognize the game, although made another "Epic Launcher" in Steam. Likely due to changing the target and basically recreated the win32 one again.

Also the manual scanned game that was added on Steam wouldn't launch. Although if I went into Epic win64 version and launched from inside the launcher, the game did seem to launch then.

**Comment by saschanaz** [comment]
Any idea why we can't use win32 binary though?

**Comment by moraroy** [comment]
@kuzeljko @Cevap05 @saschanaz  the epic update seems to bugged a bit but i have good news i managed to force the update for fresh install people. Simply delete all files here except for " EpicGamesLauncher.exe" : /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/        this forced the update and fixed the boot loop on mine, but installing Epic Online Services is still a problem, many EOS files are missing like EpicServicesOnline.msi or the EpicServicesOnlineHelper.exe... EOS wont install on "win64" binary either but at least forcing the update is possible for the NSL shortcuts for now

**Comment by Cevap05** [comment]
@moraroy Hey, just so I understand. Are you saying this method will force the update via Win32, that way NSL shortcuts to Steam can be used? How I have it setup at the moment with Win64 does not allow me too use the shortcuts, well they don't work when I scan and try to use them. If that's the case, you're saying to delete everything under NSL folder regarding Epic except for:

/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/   ?

And then just re-install Epic again? Just so I know the steps if you can clarify? Thanks

**Comment by moraroy** [comment]
@Cevap05 no don't uninstall or reinstall...just simply go into that folder path the, win32 folder. And delete everything in it except the "EpicGamesLaunchers.exe" this will fix the update loop and force it. That's all. That way the original NSL shortcuts can be used.

**Comment by moraroy** [comment]
@Cevap05 if you already bypassed the update loop by using g the win64 fix then you should be able to go back to using win32 in the shortcut if I'm not mistaken

**Comment by hys17** [comment]
Came here to say thank you for this solution! Been trying reinstalling Epic via NSL, wiping NSL and reinstalling everything...

Wish I've seen this post sooner... 

> [@Cevap05](https://github.com/Cevap05) no don't uninstall or reinstall...just simply go into that folder path the, win32 folder. And delete everything in it except the "EpicGamesLaunchers.exe" this will fix the update loop and force it. That's all. That way the original NSL shortcuts can be used.

**Comment by Cevap05** [comment]
@moraroy Thanks for the reply. So I ended up deleting the Win64 shortcut variant I made and just remade the NSL shortcut for Epic (which has Win32 in the target), and it seemed to open fine.

Although I'm having an issue with NSL shortcuts regarding Epic games, or so I thought. I had been trying to play Sifu that was free on Steam but it seems the NSL shortcut just won't work

I thought it was an issue in general, so I tried to install Fall Guys and generated a shortcut. Seemed to work fine.

I even re-installed Sifu, but same thing. It just tries to open for a few seconds and then closes. Although the game will actually run if I first open Epic Launcher, then launch from in there.

I wondered if it had anything to do with the settings for the shortcut, so I'm going to paste both Fall Guys (which works) and Sifu's. Maybe you can tell me if something is going on here? Thanks



///Fall Guys///
Target: "/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/EpicGamesLauncher.exe"

Start In: "/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/"

Launch Options: STEAM_COMPAT_DATA_PATH="/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/" %command% -'com.epicgames.launcher://apps/0a2d9f6403244d12969e11da6713137b?action=launch&silent=true'
----------------------------
///Sifu///
Target: "/home/deck/bin/umu-run" "/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/EpicGamesLauncher.exe"

Start In: "/home/deck/bin/"

Launch Options: STEAM_COMPAT_DATA_PATH="/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/" WINEPREFIX="/home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx" GAMEID="2138710" PROTONPATH="/home/deck/.steam/root/compatibilitytools.d/GE-Proton9-25" %command% -'com.epicgames.launcher://apps/d36336f190094951873ed6138ac208d8?action=launch&silent=true'

**Comment by moraroy** [comment]
@Cevap05 yep those all look correct, make sure you have umu at that location on your device.... also.... with umu shortcuts with NSL. Make sure that force compatability is "unchecked". There is small bug currently in the desktop version of NSL when it remakes shortcuts it ends up checkmarking force compatability which is not needed as it won't run. Basically in a nutshell Sifu is a umu based shortcut, and you just need to check and make sure your umu shortcuts are unchecked.

**Comment by Cevap05** [comment]
@moraroy Thanks for reply. Sorry if basic question but perhaps will help someone. So when I go to /home/deck/bin, there is no "umu-run". Even though the shortcut made it that path. Do I have to install umu and put it in that directory? Not sure what to do here but I believe this is where my problem is.

FYI haven't had issues with any other shortcuts or games thus far.

**Comment by moraroy** [comment]
@Cevap05 oh dang ok that's why it's not working, are you on bazzite? Or steamos? Desktop or decky loader plugin plugin?

**Comment by Cevap05** [comment]
@moraroy SteamOS and have/can do both Desktop or Decky plugin.

I noticed there is a directory /home/deck/.local/share/umu/ that has UMU related files. Although I don't see "umu-run" in that folder. Idk if that helps? And I believe this is installed from Steam since I didn't install it.

**Comment by moraroy** [comment]
@Cevap05 ah I see ok thanks no, it's supposed to be installed from NSL. It seems the installation of UMU by NSL has failed, it simply didn't extract properly, you can fix this yourself in the meantime if you want. Simply download the files from the umu githhub and and extract and place them there and set them to executable. Then sifu should run.

**Comment by moraroy** [comment]
@Cevap05 ah I see ok thanks no, it's supposed to be installed from NSL. It seems the installation of UMU by NSL has failed, it simply didn't extract properly, you can fix this yourself in the meantime if you want. Simply download the files from the umu githhub and and extract and place them there and set them to executable. Then sifu should run.@Cevap05 to be honest when was the last time you ran NSL? Like install a launcher because I thought I fixed that bug a couple releases ago 😅

**Comment by moraroy** [comment]
@Cevap05 just tested actually and yes you are correct it is broken again 😢 ill try to do a quick fix for that in a couple hours, it looks the curl is failing

**Comment by Cevap05** [comment]
@moraroy That actually did fix it, thank you very much! Was going to say, have used it pretty frequently past two weeks. Wasn't sure if it was me though. I've honestly had no issues with anything until trying Epic haha. First it was the update issues, then a game worked and the other didn't. At least seems like each issue has been resolved fairly quickly. Thank you for this awesome project, and your help!

**Comment by moraroy** [comment]
@Cevap05 no worries! Thank you for being patient, I'll try to fix these small issues in the next update!

**Comment by saschanaz** [comment]
> [@kuzeljko](https://github.com/kuzeljko) [@Cevap05](https://github.com/Cevap05) [@saschanaz](https://github.com/saschanaz) the epic update seems to bugged a bit but i have good news i managed to force the update for fresh install people. Simply delete all files here except for " EpicGamesLauncher.exe" : /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Epic Games/Launcher/Portal/Binaries/Win32/ this forced the update and fixed the boot loop on mine, but installing Epic Online Services is still a problem, many EOS files are missing like EpicServicesOnline.msi or the EpicServicesOnlineHelper.exe... EOS wont install on "win64" binary either but at least forcing the update is possible for the NSL shortcuts for now

Interesting, I didn't have Win64 and doing this gave me Win64. (The updater just silently went away and then kept running in background, installing things)

**Comment by hys17** [comment]
Anyone having the same issue again? Epic recently had another update and it's crashing the app, even after I did the cleanup of ".../Portal/Binaries/Win32/" folder. 

Also, I've noticed some of newly installed games cannot run with the shortcut alone (it'd pop up saying I don't own this game), but works fine if I start it directly within Epic launcher. Maybe a separate issue but I wonder if I should just wipe it and start all over again... 

If anyone else has the same issues I'd love to hear your thoughts.

**Comment by moraroy** [comment]
@hys17 yes same here the new Epic update messed me up, for me it said I couldn't install any new games as my games folder was error ing out. What I did personally in the mean time was uninstall EpIc from NSL, and then I had to delete my old games folder with my games :( in order for Epic to even fix itself, in a nutshell I had to wipe everything from Epic in the prefix. Before you end up doing that you could try a couple things... in the %appdata% folder there is more Epic folders that have to do with your login info and logs, you could try the win32 trick and then also just delete those %appdata% Epic folders and see of signing back in will help you refresh the games launcher. If that still doesn't work, as of right now I do t have a fix :( .... thankfully though, if you've run the script recently, or used the "manual scan" in the decky plugin version, all of your games and game saves are backed up in /home/deck/NSLGamesSaves if you want to inject all of your games saves back into Epic once you re-download and fix the launcher again you can do it in one go. This is a feature of the decky plugin version called "Restore Game Saves" it will put all your saves back inside of your games. To do this on the Dssktop Version you need to run the command line in konsole because I didn't make a button for it yet 😅  here is the command if you need it : /bin/bash -c 'curl -Ls https://raw.githubusercontent.com/moraroy/NonSteamLaunchers-On-Steam-Deck/main/NonSteamLaunchers.sh | nohup /bin/bash -s -- "Epic Games"'    i haven't checked on this command in a while so I believe it is still working for desktop version if you or anyone needed it.

**Comment by hys17** [comment]
> [@hys17](https://github.com/hys17) yes same here the new Epic update messed me up, for me it said I couldn't install any new games as my games folder was error ing out. What I did personally in the mean time was uninstall EpIc from NSL, and then I had to delete my old games folder with my games :( in order for Epic to even fix itself, in a nutshell I had to wipe everything from Epic in the prefix. Before you end up doing that you could try a couple things... in the %appdata% folder there is more Epic folders that have to do with your login info and logs, you could try the win32 trick and then also just delete those %appdata% Epic folders and see of signing back in will help you refresh the games launcher. If that still doesn't work, as of right now I do t have a fix :( .... thankfully though, if you've run the script recently, or used the "manual scan" in the decky plugin version, all of your games and game saves are backed up in /home/deck/NSLGamesSaves if you want to inject all of your games saves back into Epic once you re-download and fix the launcher again you can do it in one go. This is a feature of the decky plugin version called "Restore Game Saves" it will put all your saves back inside of your games. To do this on the Dssktop Version you need to run the command line in konsole because I didn't make a button for it yet 😅 here is the command if you need it: /bin/bash -c 'curl -Ls https://raw.githubusercontent.com/moraroy/NonSteamLaunchers-On-Steam-Deck/main/NonSteamLaunchers.sh | nohup /bin/bash -s -- "NSLGameSaves"' i haven't checked on this command in a while so I believe it is still working for desktop version if you or anyone needed it.

Thanks so much for such detailed response.

I've tried deleting the %appdata% folders and it didn't help unfortunately.

So I did total wipe and the deleted all the game folders that I could find. (don't care about saves so much at this point but will get to that once everything is working again). Updated NSL plugin and re-installed EGS.

This time, the epic game launcher is up to date. 
However, the issue with "I don't own this game" persists (redownloaded a couple of games to try). Am I missing anything?

Thanks.

**Comment by moraroy** [comment]
> This time, the epic game launcher is up to date. However, the issue with "I don't own this game" persists (redownloaded a couple of games to try). Am I missing anything?

nice! was it able to also install the Epic Online Services too? or did that still fail? i actually havnt come across that error yet all my games seem to install. You not owning the game if you truly do own it ,i would chalk it up still to the new updates epic has been pushing out hopefully over time through proton and their updates we can see if a fix  is made, sorry this isnt as smooth as before  btw i know its annoying.

**Comment by moraroy** [comment]
this also helps after installation and in update loop.still no EOS... run in konsole 

```
rsync -av --progress \
  /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/ProgramData/Epic/EpicGamesLauncher/Data/Update/Install/Engine/ \
  /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program\ Files\ \(x86\)/Epic\ Games/Launcher/Engine/


rsync -av --progress \
  /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/ProgramData/Epic/EpicGamesLauncher/Data/Update/Install/Portal/ \
  /home/deck/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program\ Files\ \(x86\)/Epic\ Games/Launcher/Portal/

**Comment by hys17** [comment]
> > This time, the epic game launcher is up to date. However, the issue with "I don't own this game" persists (redownloaded a couple of games to try). Am I missing anything?
> 
> nice! was it able to also install the Epic Online Services too? or did that still fail? i actually havnt come across that error yet all my games seem to install. You not owning the game if you truly do own it ,i would chalk it up still to the new updates epic has been pushing out hopefully over time through proton and their updates we can see if a fix is made, sorry this isnt as smooth as before btw i know its annoying.

EOS still fails as previously. :(
Thanks and I hope this would change soon too.

**Comment by nicosuchtet** [comment]
I also had the problem with the Epic Games Launcher update loop. After deleting the contents of the win32 folder (except for EpicGamesLauncher.exe), at least the launcher started. For the Epic Online Services, I found the following workaround:

-  Download Fall Guys from the Epic Games Store (free, ~7GB).

-  Open the following path in the Epic Games Launcher prefix: drive_c/Program Files/Epic Games/FallGuys/EpicOnlineServices.

-   Add the file EpicOnlineServicesInstaller.exe to Steam.

-   Open the properties of the Epic Games Launcher in Steam.

-   Copy the launch options (for the correct prefix).

-   Open the properties of the newly added EOS installer.

-   Paste the previously copied launch options.

-   Under compatibility, select the latest GE-Proton version.

-   Start the EOS installer via Steam.

-   Let the installation run through.

-   After installation, start the Epic Games Launcher.

-   EOS should then download some updates and be fully functional.

After that, the Epic Games Launcher worked on my Steam Deck without any issues.

**Comment by moraroy** [comment]
@nicosuchtet great find and great fix, would need to figure out a way to add that to the script somehow. So currently I've patched Epic games to now by pass the loop with the new updates ive been pushing feel free to give it a try, and now thanks to you, we can now get a full working launcher again!! I tried hosting that file on github but it's too big unfortunately haha

**Comment by hys17** [comment]
> [@nicosuchtet](https://github.com/nicosuchtet) great find and great fix, would need to figure out a way to add that to the script somehow. So currently I've patched Epic games to now by pass the loop with the new updates ive been pushing feel free to give it a try, and now thanks to you, we can now get a full working launcher again!! I tried hosting that file on github but it's too big unfortunately haha

Thanks for the new version. It's working much better now (although the "you don't own this game error" had come up a couple of times but I haven't found the pattern yet). Will keep an eye out for the new releases.
Is there a place I could get notified for a new release?
Thanks so much for your hard work.

**Comment by moraroy** [comment]
@hys17 I finally got that error the other day haha, hopefully as Epic updates it will fix itself. Right now I noticed if there is a new Epic update and you hit restart, it doesn't restart, so what I did was simply re-install epic. That seems to work

**Comment by hys17** [comment]
> [@hys17](https://github.com/hys17) I finally got that error the other day haha, hopefully as Epic updates it will fix itself. Right now I noticed if there is a new Epic update and you hit restart, it doesn't restart, so what I did was simply re-install epic. That seems to work. For new releases just watch the github, I'll try to do more minor releases.

Cool good to know. Thanks!

**Comment by moraroy** [comment]
@hys17 there is also the new note system if you wanted to give that a try :) haha we can make notes about the launchers and keep them up to date

**Comment by Vantillana** [comment]
Hello all! I'm new on all of this 😬  Ive a Lenovo Legion Go  with steam deck, Ive installed the NonsteamLauncher. I see the paths for battlenet, roxkstar, and others in the NonSteamLauncher Window. I did it twice without and then with the "separate apps IDS..."  When I open the steam deck, I see the apps, I proceed to open Battlenet,  I linked my account, downloaded warzone (it's free) to try it. It won't open, shows error BLZBNTBNA00000012. Ive forced different Proton and still happens the same 🥲  

![Image](https://github.com/user-attachments/assets/5edcfb53-59b1-44c7-a4f0-fdeaab843131)

**Comment by saschanaz** [comment]
This issue is about Epic Game Launcher, and Battle.net issue should be filed as a separate issue to get a separate look (and to help people find it in case they have the same issue).

**Comment by Vantillana** [comment]
> This issue is about Epic Game Launcher, and Battle.net issue should be filed as a separate issue to get a separate look (and to help people find it in case they have the same issue).

Doing that now. Gracias

**Comment by moraroy** [comment]
Hey in case everyone is here for Epic, recently Epic has updated to 18.80, I can confirm it now allows you to "update" within game mode. I can't confirm yet if EOS installation is still an issue though, let me know.
